tests/unit/test_main.py

```diff
@@ -410,16 +410,16 @@ def test_load_switch_attrs(self):
             "type": "switch"
         }
 
+        self.assertEqual(len(self.napp.controller.switches), 0)
         self.napp._load_switch(dpid_b, switch_attrs)
-
         self.assertEqual(len(self.napp.controller.switches), 1)
         self.assertIn(dpid_b, self.napp.controller.switches)
 
         switch = self.napp.controller.switches[dpid_b]
         self.assertEqual(switch.id, dpid_b)
         self.assertEqual(switch.dpid, dpid_b)
         self.assertFalse(switch.is_enabled())
-        self.assertFalse(switch.is_active())
+        self.assertTrue(switch.is_active())
```

Comment:
Same here about being loaded as `active=False`

Reply:
Right. It's been fixed here https://github.com/kytos-ng/topology/commit/8e262bc050b8633ef3e27d9a3b3c774d829565e5